DDOT_module/parent_classes/marginals_analyzer_interface.py

```diff
@@ -0,0 +1,64 @@
+"""
+Created on 28.04.2021
+
+@author: fbm
+@contact: filip.b.maciejewski@gmail.com
+"""
+import abc
+
+"""
+    This file holds an interface for Marginals Analyzers. BS. 
```

Comment:
Although I admire the sentiment, do please remove BS before publishing this as a package, as it may confuse some users :P